resharper/resharper-unity/src/Rider/UnitTesting/RunViaUnityEditorStrategy.cs

```diff
@@ -225,8 +228,9 @@ private void RefreshAndRunTask(IUnitTestRun run, TaskCompletionSource<bool> tcs,
                             clientControllerInfo.ExtraDependencies?.ToList(),
                             clientControllerInfo.TypeName);
 
-                    var mode = TestMode.Edit;
                     var frontendBackendModel = mySolution.GetProtocolSolution().GetFrontendBackendModel();
+                    myUsageStatistics.TrackActivity("UnitTestPreference", frontendBackendModel.UnitTestPreference.Value.ToString());
```

Comment:
@van800 Should this ID have some kind of Unity prefix on it, to distinguish from non-Unity statistics?

Reply:
I would add, thanks!

Reply:
https://github.com/JetBrains/resharper-unity/commit/10565ca3f8b222dd535f636ca0891d288a242971